client/src/views/WizardSteps/ValidateDataset.vue

```diff
@@ -39,18 +50,19 @@
           best describes your dataset?</span
         >
         <v-radio-group
+            data-test="radioBestDescribes"
             :disabled="radioBestDescribesShouldBeDisabled"
             v-model="radioBestDescribes"
             class="pl-2"
+            v-on:change="saveUserInput"
         >
           <RadioItem
               label="Public information."
               value="public"
-              :on="{
```

Comment:
- Choosing "Public Information" should launch the modal box with the title "The dataset should not be used in DPcreator". 
   - This has been disabled
- Similarly, the last choice, "Information that **would cause severe harm**", should also launch the same modal.

Reply:
ok, will check in a fix for this